### DISCUSSION TITLE
Operator in TypeScript

### DISCUSSION THREAD

**Discussion by sajjadarashhh** [dotnet/csharplang#8145] [Language Ideas]
this might be good idea to have operators for checking nullability and empty values in C# and work if values is not null or empty
with this operator something like this:

```
int a=10,b=15;

b&&a&&var s = b+a;
```
this must be helpful in blazor and maui

**Comment by HaloFour** [comment]
What does `b&&a&&var s = b+a;` mean?  I assume that expands out to `b && a && var s = b + a;` but I don't understand how the comparison and the assignment are related.

More generally, it's quite intentional that C# does not permit `int` to behave implicitly as a `bool` with `0` being `false` as "truthiness" is considered a source of bugs.
